.storybook/themes/commerce.scss

```diff
@@ -209,15 +209,21 @@
 	$tokens: (
 		default: (
 			components: (
+				attribution: (
+					color: var(--text-color),
+					font-size: var(--body-font-size-small),
```

Comment:
Is this attribution meant to scale down to 12px on mobile? I'm not sure based on the designs

Looks like it's always 14px in the designs. 
<img width="883" alt="Screen Shot 2022-05-06 at 11 00 20" src="https://user-images.githubusercontent.com/5950956/167169680-b75cdb5a-1f4c-4270-b1be-62a64d87824c.png">
 
<img width="150" alt="Screen Shot 2022-05-06 at 10 59 23" src="https://user-images.githubusercontent.com/5950956/167169710-7ad7b26a-dc13-468f-baf4-456f8825c2b7.png">

Reply:
I'm going to discuss this w/ Kelly very shortly
I may need to change both Attribution and Date.
Will follow-up! Thanks!

Reply:
Hey! So let's make sure to bring this up in design sync on Monday.

It seems like we do not have have figma illustrations for all mobile vs desktop designs yet, which is cool, but without them we don't know when the components should vary based on breakpoint or when they should be the same. In lieu of that, we have two options:

- Assume mobile and desktop are the same unless explicitly illustrated in figma. This means in Figma we would paying more attention to the typography *size and line-height* attributes instead of the typography *AG* key which denotes a token. In code, we would ignore our body responsive alias tokens in favor of explicit global tokens.
- Assume the we should be using responsive values unless explicitly illustrated in figma. This means in Figma we would paying more attention to the typography *AG* key which denotes a token instead of the typography *size and line-height* attributes. In terms of code in this case we should use our alias tokens instead of global tokens.

Which direction should we go? Either way, we might need a desktop specific follow up ticket to do any clean up afterwards.

Reply:
@JackHowa : Following up.  As per design sync meeting and Robin's notes above, the Pill component will use global tokens for typography that will apply to all breakpoints. Thanks!